astropy/cosmology/io/tests/test_model.py

```diff
@@ -164,7 +164,7 @@ def test_is_equivalent_to_model(self, cosmo, method_name, to_format, format):
         assert not isinstance(obj, Cosmology)
 
         is_equiv = cosmo.is_equivalent(obj, format=format)
-        assert is_equiv is (True if format is not False else False)
+        assert is_equiv is (bool(format is not False))
```

Comment:
Strange that here it has the `bool` version but above it does not. And quite a few more.  I think the `bool` can be omitted.

Reply:
I believe that is because of https://github.com/astropy/astropy/pull/14274#discussion_r1071743807. I just did not go through the rest of the proposed changes to comment the same thing on identical situations.

Reply:
✅ Done.